diracx-logic/src/diracx/logic/auth/token.py

```diff
@@ -337,6 +338,17 @@ def create_token(payload: TokenPayload, settings: AuthSettings) -> str:
     return encoded_jwt.decode("ascii")
 
 
+def read_token(payload: str, settings: AuthSettings) -> dict:
```

Comment:
I think there is already a similar function named `verify_dirac_refresh_token`, can't you just reuse it?